wykop_scrapy_vol2.py

```diff
@@ -116,20 +114,20 @@ def checking_comments():
         print("Brak komentarzy")
 
 
+data_base_links = pd.read_csv("post_links.csv")
```

Comment:
'data_base_links' maybe better name is simply 'post_links'?

Reply:
Will be changed in next commit.